api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryService.java

```diff
@@ -148,5 +148,9 @@ default <T> Stream<List<T>> getBatchedStream(BiFunction<Long, Long, List<T>> get
 
   int getWorkspaceCount();
 
-  List<ReportingLeonardoAppUsage> getLeonardoAppUsage();
+  List<ReportingLeonardoAppUsage> getLeonardoAppUsage(long limit, long offset);
+
+  default Stream<List<ReportingLeonardoAppUsage>> getBatchedLeonardoAppUsageStream() {
+    return getBatchedStream(this::getLeonardoAppUsage);
+  }
 }
```

Comment:
all existing streaming is streamqing query from mysql while app usage is BigQuery. 
Does BQ support this?

Reply:
I don't see why it wouldn't as long as `getLeonardoAppUsage` returns the same type